pkg/sql/logictest/testdata/logic_test/alter_table

```diff
@@ -2664,3 +2664,11 @@ ALTER TABLE t81448 ADD COLUMN b INT PRIMARY KEY
 
 statement ok
 DROP TABLE t81448
+
+subtest add_column_non_indexable_type
+
+statement ok
+CREATE TABLE t1_non_indexable(n int);
```

Comment:
nit: pipe this statement and the one below through sqlfum.pt for nicer formatting